en/django_start_project/README.md

```diff
@@ -118,6 +118,35 @@ ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
 
 > Also add `.amazonaws.com` to the `ALLOWED_HOSTS` if you are using cloud9
 
+> If you are hosting your project on `Glitch.com`, let us protect the Django secret key that needs to 
+> remain confidential (otherwise, anyone remixing your project could see it) :
+>   * First, we are going to create a random secret key. Open the Glitch terminal again, and type the following command :
```

Comment:
The rest of your contribution has each sentence on a source line of its own, which, although most of the Django Girls tutorial isn't written that way, is IMO a good idea. (It's essentially the most basic form of ["semantic linefeeds"](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) a.k.a. ["semantic line breaks"](https://sembr.org/).) Let's do apply that here, too.

Putting a space before a colon (`:`) is AFAIK a French convention not usually followed in English.

```suggestion
>   * First, we are going to create a random secret key.
>     Open the Glitch terminal again, and type the following command:
```